pkg/operator/ceph/cluster/cluster.go

```diff
@@ -352,3 +372,49 @@ func versionSupported(version string) bool {
 	}
 	return false
 }
+
+func (c *cluster) setOrchestrationRunning(yesNo bool) {
+	c.orchRunMux.Lock()
+	c.orchestrationRunning = yesNo
+	c.orchRunMux.Unlock()
+}
+
+func (c *cluster) setOrchestrationPending(yesNo bool) {
```

Comment:
Similarly, this is only called with `true`. Shall we rename it?

Reply:
OK. Will change it accordingly!